extensions/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Added
+
+- `packageCheck` function to check a package against the default policy
```

Comment:
I think this is honestly kinda misleading. This sounds just like `phylum package` but it's not to check a package against the default policy, but to check **a list** of packages against the default policy.

Do we maybe want to rename this to `checkPackages`? I know the endpoint is `packages/check`, but I think having the verb first is more natural for function names and `packagesCheck` doesn't sound as good.

Reply:
I definitely meant to write "check a list of packages" and not "check a package".

And yes, `checkPackages` is a better name for the function. I'll make that change.